### DISCUSSION TITLE
Cloud save? Internet access? code/word-splitter (not sure if this make sense) and file interactions?

### DISCUSSION THREAD

**Discussion by whizshounen** [ChatGPTNextWeb/NextChat#662] [Ideas]
Just some ideas its alright if these are ignored. I love you work its clean and easy to setup and I've been using it everyday just fine. Just some ideas I thought would be nice to have:-

1. **Cloud save**
for previous threads. I believe for now the previous conversations are only saved on the respective machine when you use it.

2. **Internet access**
Like BingChat I guess. Not sure if there is a better way other than "User prompt" > chatgpt > google search > responds send back to chat gpt > answer send to user

3. **Cod/WordSplitter**
I have no idea if this make sense but as far as I know gpt3.5 model are limited to 4000 tokens. So if you try to send a 10,000 lines of codes or words it will just not be responsive. When I asked chatgpt how to go around this it says to use this script that it made to split the code and have it sent to the model. 

So.. if I understand correctly we can have a button in the chatroom at the text box for input where when we click a pop up will appear and we put the 10,000 code inside and click submit on the pop up box and it will be able to split the code and send it to the model in batch? I dont know can just ignore this if it useless.

4. **File interactions**

something like this - https://github.com/guangzhengli/ChatFiles

**Comment by Yidadaa** [comment]
1 and 4 are in plan.
